test/e2e/view-caseload-capacity.js

```diff
@@ -0,0 +1,50 @@
+const expect = require('chai').expect
+
+const workloadCapacityHelper = require('../helpers/data/workload-capacity-helper')
+
+var workloadOwnerIds = []
+
+describe('View your caseload capacity flow', () => {
+  before(function () {
+    return workloadCapacityHelper.selectIdsForWorkloadOwner()
+      .then(function (results) {
+        workloadOwnerIds = results
+      })
+  })
+
+  it('should navigate to the workload owner caseload capacity screen', () => {
+    return browser.url('/offender-manager/' + workloadOwnerIds.filter((item) => item.table === 'workload_owner')[0].id + '/caseload-capacity')
+      .waitForExist('.plot-container.plotly')
+      .waitForExist('.heading-secondary')
+      .getValue('.heading-secondary', function (title) {
+        expect(title).to.contains('Offender Manager Capacity')
```

Comment:
Looks like the new subtitle is just the org unit, i.e 'Offender Manager' as opposed to 'Offender Manager Capactity'. Same for tests below